rfcs/text/0000-data-citation-plan.md

```diff
@@ -0,0 +1,131 @@
+### DCP PR:
+
+***Leave this blank until the RFC is approved** then the **Author(s)** must create a link between the assigned RFC number and this pull request in the format:*
+
+`[dcp-community/rfc#](https://github.com/HumanCellAtlas/dcp-community/pull/<PR#>)`
+
+# HCA DCP Data Citation Plan
+
+## Summary
+
+Data Contributors need to be able to cite data sets that they have contributed to the DCP.
+Data Consumers need to be able to cite the DCP data that they have used in their research projects.
+Providing citable records for the DCP will enable all contributors and consumers to reference the data stored in the DCP that they have used in their scientific publications.
+
+## Author(s)
+
+ [Trevor Heathorn](mailto:theathor@ucsc.edu)
+
+## Shepherd
+
+ [Trevor Heathorn](mailto:theathor@ucsc.edu)
+
+## Motivation
+
+There is currently no clear and agreed upon definition of the requirements for Data Citation in the DCP.
+Key issues that this RFC seeks to resolve are:
+  - What is the minimum feature set required for a first release (Phase 1)?
+  - What are the discrete set of features that make sense for subsequent releases (e.g. Phase 2, Phase 3, etc.)
+  - In which phase, if any, is support for a formal digital object identifier ([DOI](https://en.wikipedia.org/wiki/Digital_object_identifier)) required?
+
+### User Stories
+
+1. As a data contributor (e.g. researcher with a pipette), it is essential to have a unique way for my project (consisting of primary and secondary analysis data files and associated metadata files) to be identified so that others can properly cite my work when using my data, and that this citation identifier is available in the DCP Data Portal. Note: This will discourage but not prevent others from using my data without attribution.
+2. As a data consumer (e.g. researcher with a keyboard), I want to be able to view and share a unique citation identifier so that a reader of my manuscript can obtain the data needed to reproduce my results. Anyone can use the citation identifier to view and download all the original cited data and metadata files for a project from the DCP.
+3. As a data consumer, I need a simple way to reference a project in the DCP so that I can fulfill the requirements of a Creative Commons attribution license (CC-BY).
+4. As a data contributor or consumer, I need a way to use the citation identifier to access the output produced by the DCP Matrix Service for the data being cited.
+5. As a member of the Data Operations team I want to be able to create citations for each of the projects that I include in a Data Release (which is defined as a point-in-time selection of specific verions of data from the DCP).
```

Comment:
I think one should be able to create a citation for each project, regardless of inclusion of a Data Distribution. So maybe this is better...
>As a data consumer, I am going to publish research based on HCA data and I want to create a citation that references the specific version of each project that I used.

Reply:
...and just read User Story 6, so I think that covers this. I change my suggestion to remove 5, fix the typo in 6 ('contributor') and remove 'or data consumer' in 6 (a data consumer updating a project is not on the table)

Reply:
...reading further still, I see that a 'research project' in 6 actually seems to refer to a 'data consumer-curated collection of HCA data'
I think my suggested user story above covers both cases of DataOps cutting a Data Distribution and a consumer making their own collection. If each version of each project can be referenced, then any collection is just a list of those references, right?
Unless there is going to be a difference between a collection made by someone DCP internal vs external

Reply:
The implementation mechanism for User Stories 5 and 6 may differ. We may not require generic Data Portal login for the Data Ops team (e.g. by supplying specific internal tools to them) and the level of citation granularity may be restricted (e.g. to whole projects and/or the whole Data Distribution). User Story 6 allow for a wide audience of authorized users to create arbitrary cross-project citations.

Reply:
Similar with my comment below, I think assumptions about future functionality is introducing unnecessary complications (Data Distributions, DataOps logins, non-DCP user logins, collections). For instance, it has not been decided that all Data Distributions reference data at the 'project' level. 
Would a more straightforward (and thus, easier to spec out & implement) proposal be Phase 1: Stable non-versioned citable project URLs & Phase 2: Stable citable project _version_ URLs? It seems to me that a project version should be citable independent of any collection/distribution, so once Phase 2 is complete, those project versions could easily be grouped/listed in whatever packaging is required, whether by DCPers or non-DCPers.

Reply:
My intention was to state that for Phase 2 the Data Ops team decides when a project version becomes citable and creates the citable reference (DOI) for each version. Phase 3 then allows external users to create their own citations for any arbitrary set of data, I've attempted to remove any dependence on anything but the most  basic Data Distribution features.

Reply:
@theathorn I think that if we automatically make the headline url for a project citable then we should also automate citability of distinct project versions when we support access at that level regardless of why the version was updated, it will make the whole process much simpler and much more FAIR

@jahilton and the Data Ops team will, of course, end up needing to make decisions about how releases are updated and versioned but I agree with @jahilton that being too specific on mechanisms around that here is a bad idea

Reply:
Removed all references to Data Distributions and replaced with "discrete project versions".